packages/cli/src/commands/shares/delete.ts

```diff
@@ -0,0 +1,43 @@
+import { ShareService } from '@meeco/sdk';
+import { AuthConfig } from '../../configs/auth-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class SharesDelete extends MeecoCommand {
+  static description =
+    'Delete a share. Both the owner of the shared data and the recipient of the share can delete the share';
+
+  static flags = {
+    ...MeecoCommand.flags,
+    ...authFlags
+  };
+
+  static args = [
+    {
+      name: 'itemId',
```

Comment:
This should be `shareId`? Or is this the item id? The docs seem to be a bit conflucting.

Reply:
I think it should be share_id, I kept it the same as shares: get command. I will change it over there as well.